### PR TITLE
add LeNER-Br dataset

### PR DESCRIPTION
**Pull request by jonatasgrosman**
Adding the LeNER-Br dataset, a Portuguese language dataset for named entity recognition

### CODE REVIEW

**Comment by jonatasgrosman** [comment]
I don't know what happened, my first commit passed on all checks, but after just a README.md update one of the scripts failed, is it normal? 😕

**Comment by TevenLeScao** [comment]
Looks like a flaky connection error, I've launched a re-run, it should be fine :)

File: datasets/lener_br/lener_br.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+_HOMEPAGE = "https://cic.unb.br/~teodecampos/LeNER-Br/"
+
+_URL = "https://github.com/peluz/lener-br/raw/master/leNER-Br/"
+_TRAINING_FILE = "train/train.conll"
+_DEV_FILE = "dev/dev.conll"
+_TEST_FILE = "test/test.conll"
+
+
+class LeNERBr(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/lener_br/lener_br.py:62]:
```suggestion
class LenerBr(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `lener_br`

File: datasets/lener_br/dataset_infos.json [hunk 2]
```diff
@@ -0,0 +1 @@
...ype": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "le_ner_br", "config_name": "lener_br", "version": {"version_str": "1.0.0", "description": null, "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 3984189, "num_examples": 7828, "dataset_name": "le_ner_br"}, "validation": {"name": "validation", "num_bytes": 719433, "num_examples": 1177, "dataset_name": "le_ner_br"}, "test": {"name": "test", "num_bytes": 823708, "num_examples": 1390, "dataset_name": "le_ner_br"}}, "download_checksums": {"https://github.com/peluz/lener-br/raw/master/leNER-Br/train/train.conll": {"num_bytes": 2142199, "checksum": "6fdf9066333c84565f9e3d28ee8f0f519336bece69b63f8d78b8de0fe96dcd47"}, "https://github.com/peluz/lener-br/raw/master/leNER-Br/dev/dev.conll": {"num_bytes": 402497, "checksum": "7e350feb828198031e57c21d6aadbf8dac92b19a684e45d7081c6cb491e2063b"}, "https://github.com/peluz/lener-br/raw/master/leNER-Br/test/test.conll": {"num_bytes": 438441, "checksum": "f90cd26a31afc2d1f132c4473d40c26d2283a98b374025fa5b5985b723dce825"}}, "download_size": 2983137, "post_processing_size": null, "dataset_size": 5527330, "size_in_bytes": 8510467}}
```
Comment (lhoestq) [inline datasets/lener_br/dataset_infos.json:1]:
```suggestion
{"lener_br": {"description": "\nLeNER-Br is a Portuguese language dataset for named entity recognition \napplied to legal documents. LeNER-Br consists entirely of manually annotated \nlegislation and legal cases texts and contains tags for persons, locations, \ntime entities, organizations, legislation and legal cases.\nTo compose the dataset, 66 legal documents from several Brazilian Courts were\ncollected. Courts of superior and state levels were considered, such as Supremo\nTribunal Federal, Superior Tribunal de Justi\u00e7a, Tribunal de Justi\u00e7a de Minas\nGerais and Tribunal de Contas da Uni\u00e3o. In addition, four legislation documents\nwere collected, such as \"Lei Maria da Penha\", giving a total of 70 documents\n", "citation": "\n@inproceedings{luz_etal_propor2018,\n    author = {Pedro H. {Luz de Araujo} and Te'{o}filo E. {de Campos} and\n    Renato R. R. {de Oliveira} and Matheus Stauffer and\n    Samuel Couto and Paulo Bermejo},\n    title = {{LeNER-Br}: a Dataset for Named Entity Recognition in {Brazilian} Legal Text},\n    booktitle = {International Conference on the Computational Processing of Portuguese ({PROPOR})},\n    publisher = {Springer},\n    series = {Lecture Notes on Computer Science ({LNCS})},\n    pages = {313--323},\n    year = {2018},\n    month = {September 24-26},\n    address = {Canela, RS, Brazil},\t  \n    doi = {10.1007/978-3-319-99722-3_32},\n    url = {https://cic.unb.br/~teodecampos/LeNER-Br/},\n}\n", "homepage": "https://cic.unb.br/~teodecampos/LeNER-Br/", "license": "", "features": {"id": {"dtype": "string", "id": null, "_type": "Value"}, "tokens": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "ner_tags": {"feature": {"num_classes": 13, "names": ["O", "B-ORGANIZACAO", "I-ORGANIZACAO", "B-PESSOA", "I-PESSOA", "B-TEMPO", "I-TEMPO", "B-LOCAL", "I-LOCAL", "B-LEGISLACAO", "I-LEGISLACAO", "B-JURISPRUDENCIA", "I-JURISPRUDENCIA"], "names_file": null, "id": null, "_type": "ClassLabel"}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "lener_br", "config_name": "lener_br", "version": {"version_str": "1.0.0", "description": null, "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 3984189, "num_examples": 7828, "dataset_name": "lener_br"}, "validation": {"name": "validation", "num_bytes": 719433, "num_examples": 1177, "dataset_name": "lener_br"}, "test": {"name": "test", "num_bytes": 823708, "num_examples": 1390, "dataset_name": "lener_br"}}, "download_checksums": {"https://github.com/peluz/lener-br/raw/master/leNER-Br/train/train.conll": {"num_bytes": 2142199, "checksum": "6fdf9066333c84565f9e3d28ee8f0f519336bece69b63f8d78b8de0fe96dcd47"}, "https://github.com/peluz/lener-br/raw/master/leNER-Br/dev/dev.conll": {"num_bytes": 402497, "checksum": "7e350feb828198031e57c21d6aadbf8dac92b19a684e45d7081c6cb491e2063b"}, "https://github.com/peluz/lener-br/raw/master/leNER-Br/test/test.conll": {"num_bytes": 438441, "checksum": "f90cd26a31afc2d1f132c4473d40c26d2283a98b374025fa5b5985b723dce825"}}, "download_size": 2983137, "post_processing_size": null, "dataset_size": 5527330, "size_in_bytes": 8510467}}
```

`le_ner_br`-> `lener_br`

**Review by lhoestq — APPROVED** [review]
Nice thank you !

**Comment by lhoestq** [comment]
The RemoteDatasetTest error in the CI is just a connection error, we can ignore it

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**